theme/static/js/collaborate.js

```diff
@@ -85,20 +84,30 @@ $(document).on('change', '.btn-file :file', function () {
 });
 
 $(document).ready(function () {
+    set_feedback_visible(false);
     $(".btn-ask-to-join").click(request_join_group_ajax_submit);
     $(".btn-act-on-request").click(act_on_request_ajax_submit);
     $("#txt-search-groups").keyup(function () {
+        set_feedback_visible(false);
+        let num_match_found = 0;
```

Comment:
This would be cleaner as a boolean since we don't really care about the number of occurrences, but rather whether there is any or not.

We would just need to initiate it as `false`.